code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -483,6 +538,15 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/gun/energy/taser/mounted/cyborg(src)
 	src.modules += new /obj/item/taperoll/police(src)
 	src.modules += new /obj/item/device/holowarrant(src)
+	src.modules += new /obj/item/weapon/pen/robopen(src) // to rwite on a peper
+	src.modules += new /obj/item/weapon/form_printer(src) // paper dispenser
+	src.modules += new /obj/item/weapon/gripper/paperwork(src) // to transfer a paper around
+	src.modules += new /obj/item/weapon/hand_labeler(src) // for marking some precious things
+	src.modules += new /obj/item/weapon/folder/blue(src) // for any remarks
+	src.modules += new /obj/item/weapon/handcuffs/ziptie(src) // for tie..tye..tieyeing people

```

Comment:
I would prefer the zipties to the handcuffs, because right now borgs are infinite cuff dispencers and they dont have the ability to pick them up again